client/source/components/questionsAnswers/QAItem.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled from 'styled-components';
+import AnswerList from './AnswerList';
+import HelpfulQ from './HelpfulQ';
+
+const QAItem = function CreateQAItemComponent({ id, question, getQuestions }) {
+  const {
+    question_body: questionBody,
+    answers,
+    question_helpfulness: questionHelpfulness
+  } = question;
+
+  return (
+    <ListItem>
+      <QuestionRow>
+        <Question>
+          <H3>Q:</H3>
```

Comment:
I'm guessing you might still be working through styling, so just a comment as a reminder.

Line 17 - visually from a styling perspective, this appears to have a different color from the rest of the font colors. 

Line 18 - suggestion to take out <strong> element, and style it via styled components.

Reply:
I was experimenting with trying to make the question stand out a bit with the <strong> element, but our font doesn't seem to be able to be bolded. However, I don't really want it bolded anyway, so I just removed the <strong> element.

Reply:
Ah I see. For reference when you get to more styling (already looks so good!), you can explore the font-weight: bold property. I used this via styled components and it seemed to bold the font.